### ISSUE TITLE
For info - another sigrok/pico project with merge still open

### ISSUE THREAD

**Issue by tony1tf** [pico-coder/sigrok-pico#8]
Hi @pico-coder

I found another one - might be useful if you needed to exchange any ideas.

https://github.com/markb139/pico_logic

**Comment by pico-coder** [comment]
Dang, wish I'd found that before I started coding.  Having a real USB implementation might have come in handy.  

At this point  (now that the serial bugs are worked out) I'm not sure there is a large motivation to going with USB TMC (test and measurement class) rather than what I have. While the serial CDC might add an additional layer, I directly call the tud function call from the rp2040 code to send sample data.  Thus I bypass all the serial overhead in the performance critical part of the design.  The host still sees some overhead, but normally the host is much faster, and from what I've experience it seems like the sigrok driver overhead is much larger than the serial processing overhead.

I did file an issue to get a bit of discussion going
https://github.com/markb139/pico_logic/issues/9
